Description: Fix ntploggps for newer versions of gps module
 Current versions of the gps module have no isnan() function.  This has
 been removed in favor of isfinite().
 .
 Note that this change breaks ntploggps on older versions of the gps
 module.
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/863/
Origin: https://gitlab.com/NTPsec/ntpsec/commit/87ffc14e39bad36fe59f24e93bb172915f2b5e0b
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2018-12-17

--- a/ntpclients/ntploggps.py
+++ b/ntpclients/ntploggps.py
@@ -171,7 +171,7 @@
         if isinstance(t, int):
             return t
         if isinstance(t, float):
-            if gps.isnan(t):
+            if not gps.isfinite(t):
                 return None
             return t
         return gps.isotime(t)
